### DISCUSSION TITLE
how to highlight current buffer in nvimtree

### DISCUSSION THREAD

**Discussion by tydra-wang** [nvim-tree/nvim-tree.lua#1403] [Q&A] [answered]
Need help. I cannot find some thing like `highlight_focused_file` in document.

**Comment by kyazdani42** [comment, marked as answer]
`:help nvim-tree.update_focused_file`

**Reply by tydra-wang** [reply]
update_focused_file is already enabled in my config. It does un-collapse the folders recursively to find the file when I change buffer, however it's just not highlighted.

**Reply by kyazdani42** [reply]
see `:hi CursorLine` and `:help hl-CursorLine`

**Reply by tydra-wang** [reply]
It seems like ` autocmd FileType NvimTree setlocal cursorline ` cannot work

**Reply by kyazdani42** [reply]
you should override `NvimTreeCursorLine` with `:hi NvimTreeCursorLine guifg=some_color guibg=some_color` or such in your theme.

**Reply by tydra-wang** [reply]
I just fixed it by adding `setl cursorline` to my mapping right after `NvimTreeToggle`, but still wondering why `autocmd FileType` not work.

```lua
local wk = require "which-key"
wk.register {
   ["<leader>e"] = { "<cmd>NvimTreeToggle<cr><cmd>setl cursorline<cr>", "explore nvim-tree" },
}
```

**Reply by kyazdani42** [reply]
not sure why you need to do this, but still if it works for you. You could also try to do that on `BufEnter NvimTree_*`
